### PR TITLE
Feature/#197 Include ServiceWorker module in StorefrontModule

### PR DESCRIPTION
**Pull request by znikola**
Importing `ServiceWorkerModule` to `pwa.module.ts`

### CODE REVIEW

**Comment by znikola** [comment]
PR #223

File: projects/storefrontlib/src/lib/pwa/pwa.module.ts [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
 import { Config, ConfigModule } from '@spartacus/core';
+
 import { defaultPWAModuleConfig, PWAModuleConfig } from './pwa.module-config';
 import { AddToHomeScreenBtnComponent } from './add-to-home-screen-btn/add-to-home-screen-btn.component';
-import { CommonModule } from '@angular/common';
+
+export function pwaConfigurationFactory(
+  pwaConfig: PWAModuleConfig
+): RegistrationOptions {
+  return { enabled: (pwaConfig.production && pwaConfig.pwa.enabled) || false };
```
Comment (WeizhengSap) [inline projects/storefrontlib/src/lib/pwa/pwa.module.ts]: this "|| false" is not necessary.

File: projects/storefrontlib/src/lib/pwa/pwa.module.ts [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+}
 
 @NgModule({
-  imports: [CommonModule, ConfigModule.withConfig(defaultPWAModuleConfig)],
-  providers: [{ provide: PWAModuleConfig, useExisting: Config }],
+  imports: [
+    CommonModule,
+    ConfigModule.withConfig(defaultPWAModuleConfig),
+    ServiceWorkerModule.register('/ngsw-worker.js')
```
Comment (WeizhengSap) [inline projects/storefrontlib/src/lib/pwa/pwa.module.ts]: can we do this: ServiceWorkerModule.register('/ngsw-worker.js', {enabled: (Config.production && Config.pwa.enabled)}), and remove the provider for "RegistrationsOptions"?

File: projects/storefrontlib/src/lib/pwa/pwa.module.ts (hunk 1 above)
Comment (znikola) [reply projects/storefrontlib/src/lib/pwa/pwa.module.ts]: It is in a case when `pwaConfig.production`  is `undefined`

**Pull request merged**